tests/formattedcode/data/spdx/license_known/expected.rdf

```diff
@@ -1,117 +1,105 @@
-            {
-              "@rdf:resource": "http://www.spdx.org/files#SPDXRef-2"
-            },
-            {
-              "@rdf:resource": "http://www.spdx.org/files#SPDXRef-3"
-            }
-          ],
-          "@rdf:about": "http://www.spdx.org/tools#SPDXRef-Package",
-          "ns1:name": "scan"
-        }
+      "spdx:licenseConcluded": {
+        "@rdf:resource": "http://spdx.org/rdf/terms#noassertion"
+      },
+      "spdx:licenseDeclared": {
+        "@rdf:resource": "http://spdx.org/rdf/terms#noassertion"
       },
-      "ns1:referencesFile": [
+      "@rdf:about": "#SPDXRef-001",
+      "spdx:licenseInfoFromFiles": [
         {
-          "ns1:File": {
-            "ns1:licenseInfoInFile": {
-              "@rdf:resource": "http://spdx.org/licenses/Apache-2.0"
-            },
-            "ns1:licenseConcluded": {
-              "@rdf:resource": "http://spdx.org/rdf/terms#noassertion"
-            },
-            "ns1:copyrightText": {
-              "@rdf:resource": "http://spdx.org/rdf/terms#none"
-            },
-            "ns1:checksum": {
-              "ns1:Checksum": {
-                "ns1:checksumValue": "2b8b815229aa8a61e483fb4ba0588b8b6c491890",
-                "ns1:algorithm": "http://spdx.org/rdf/terms#checksumAlgorithm_sha1"
-              }
-            },
-            "ns1:fileName": "./scan/apache-2.0.LICENSE",
-            "@rdf:about": "http://www.spdx.org/files#SPDXRef-2"
-          }
+          "@rdf:resource": "http://spdx.org/licenses/Apache-2.0"
         },
         {
-          "ns1:File": {
-            "ns1:licenseInfoInFile": {
-              "@rdf:resource": "http://spdx.org/licenses/CC0-1.0"
-            },
-            "ns1:licenseConcluded": {
-              "@rdf:resource": "http://spdx.org/rdf/terms#noassertion"
-            },
-            "ns1:copyrightText": {
-              "@rdf:resource": "http://spdx.org/rdf/terms#none"
-            },
-            "ns1:checksum": {
-              "ns1:Checksum": {
-                "ns1:checksumValue": "172444e7c137eb5cd3cae530aca0879c90f7fada",
-                "ns1:algorithm": "http://spdx.org/rdf/terms#checksumAlgorithm_sha1"
-              }
-            },
-            "ns1:fileName": "./scan/cc0-1.0.LICENSE",
-            "@rdf:about": "http://www.spdx.org/files#SPDXRef-3"
-          }
+          "@rdf:resource": "http://spdx.org/licenses/CC0-1.0"
         }
       ],
-      "ns1:relationship": [
+      "spdx:packageVerificationCode": {
+        "spdx:PackageVerificationCode": {
+          "spdx:packageVerificationCodeValue": "517c26f04b5a3edaaa0c17bf1a6571953209e209"
+        }
+      },
+      "spdx:relationship": [
         {
-          "ns1:Relationship": {
-            "ns1:relationshipType": {
+          "spdx:Relationship": {
+            "spdx:relationshipType": {
               "@rdf:resource": "http://spdx.org/rdf/terms#relationshipType_contains"
             },
-            "ns1:spdxElementId": "SPDXRef-001",
```

Comment:
Where have the Ids gone? I thought these were must have data?

Reply:
There was a bug in the previous spdx-tools version that did not include the correct namespace in the SpdxIds. Now, as the document namespace includes a generated UUID, it can't be used for direct comparison.  Therefore I had to introduce the code [here](https://github.com/nexB/scancode-toolkit/pull/3456/files/26f26c5f036f0e3468d5552788216a55f042907b#diff-5d7d79946ca5faabcbfa18aead7e79acc42cb26fd70a254dd0142231d1d211e6R34-R36) to strip the namespace from the data.
Still, this will at least check that all SpdxIds have the same document namespace as they wouldn't be stripped otherwise.

Reply:
Sorry, I misunderstood your comment.
The "SPDXRef-001" is not missing, it [can be found a good deal further up](https://github.com/nexB/scancode-toolkit/pull/3456/files/26f26c5f036f0e3468d5552788216a55f042907b?diff=split&w=0#diff-bcb5f143ba2af18266072e7c8004418d08e8b57400f643fa8c623cafde1e4430R17) with the package that the relationship belongs to. This is similar to the [RDF relationship example in the spec](https://spdx.github.io/spdx-spec/v2.3/relationships-between-SPDX-elements/#1113-examples).